packages/vm/test/api/EIPs/eipUtils.ts

```diff
@@ -0,0 +1,108 @@
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
```

Comment:
Also here, renaming suggestion: just `utils.ts` (redundant, context is clear by folder naming).